### PR TITLE
feat(191) open exchange forex currency data

### PR DESCRIPTION
**Pull request by clD11**
This PR adds a new fiat price feed (fiat-oxr) which gives the user more options when getting Forex currency data.

These are the Kelp CLI changes only and an example has been provided in the sample_buysell.cfg

### CODE REVIEW

File: plugins/fiatFeedOxr.go [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+	res, err := f.client.Get(f.url)
+	if err != nil {
+		return 0, fmt.Errorf("oxr: error %w", err)
+	}
+	defer res.Body.Close()
+
+	if res.StatusCode != 200 {
+		var e oxrError
+		if err := json.NewDecoder(res.Body).Decode(&e); err != nil {
+			return 0, err
```
Comment (nikhilsaraf) [inline plugins/fiatFeedOxr.go:59]: can we wrap these errors with a message describing the nature of these errors using `fmt.Errorf()` please?
including the ones below. message can be something short like `fmt.Errorf("failed to decode error: %s", err)`

File: plugins/fiatFeedOxr.go [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+		}
+		return 0, e
+	}
+
+	var rates oxrRates
+	if err := json.NewDecoder(res.Body).Decode(&rates); err != nil {
+		return 0, err
```
Comment (nikhilsaraf) [inline plugins/fiatFeedOxr.go:66]: similar here too. we have a pattern of wrapping all errors before returning in the codebase so trying to ensure we stick to that.

File: plugins/fiatFeedOxr_test.go [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+)
+
+func Test_GetPrice_ShouldReturnZero_ClientError(t *testing.T) {
+	oxrFeed := newFiatFeedOxr(tests.RandomString())
+	price, err := oxrFeed.GetPrice()
+	assert.Equal(t, price, float64(0))
+	assert.Contains(t, err.Error(), "oxr: error ")
+}
+
+func Test_GetPrice_ShouldReturnZero_OxrError(t *testing.T) {
```
Comment (nikhilsaraf) [inline plugins/fiatFeedOxr_test.go:22]: 👍 💯

**Review by nikhilsaraf — DISMISSED** [review]
sweet, this is awesome, thank you! 
added some comments inline, but good to merge. 💯 🎉

File: examples/configs/trader/sample_buysell.cfg [hunk 4]
```diff
@@ -39,11 +39,17 @@ DATA_FEED_A_URL="kraken/XXLM/ZUSD/mid"
...
-# sample priceFeed with the "fiat" type.
-#DATA_TYPE_B="fiat"
+# Sample Price Feed (type fiat) - API Layer see https://apilayer.com/
 # you can use a service like apilayer.net to get prices for fiat if you want real-time updates. You will need to fill in the access_key in this url
+#DATA_TYPE_B="fiat"
 #DATA_FEED_B_URL="http://apilayer.net/api/live?access_key=&currencies=NGN"
 
+# Sample Price Feed (type fiat) - Fiat Open Exchange Rates see https://docs.openexchangerates.org/docs
+# To use this you must supply an app_id parameter (see https://docs.openexchangerates.org/docs/authentication)
+# OXR Free plan only allows USD as base rate and our implementation only allows one symbol e.g. STR (XLM) (see url below)
```
[the remark this answers is not in the document]
Comment (clD11) [reply examples/configs/trader/sample_buysell.cfg]: I can see how that may be confusing, I have tried to make it clearer. Basically it is -

1. The base currency cannot be controlled on a free plan and is always USD
2. Our implementation is limited to any single valid symbol for example symbols=NGN and not a list of symbols symbols=NGN,HKD

File: plugins/fiatFeedOxr.go [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+	return &fiatFeedOxr{
+		url:    url,
+		client: http.Client{Timeout: 10 * time.Second},
+	}
+}
+
+func (f *fiatFeedOxr) GetPrice() (float64, error) {
+	res, err := f.client.Get(f.url)
+	if err != nil {
+		return 0, fmt.Errorf("oxr: error %w", err)
```
[the remark this answers is not in the document]
Comment (clD11) [reply plugins/fiatFeedOxr.go:52]: It just warps the error

**Review by nikhilsaraf — APPROVED** [review]
LGTM, thanks! 🎉

**Pull request merged**